README.md

```diff
@@ -699,7 +699,7 @@ Here are the metrics settings:
 
 #### Boot-related settings
 
-*Please note that boot settings only exist for bare-metal variants at the moment*
+*Please note that boot settings only exist for the metal-k8s-1.22 variant and the k8s-1.23 (or above) variants.*
```

Comment:
`metal-k8s-1.21`'s `mod.rs` is symlinked to `metal-k8s-1.22`'s `mod.rs`
```
$ ls -al sources/models/src/metal-*
lrwxrwxrwx 1 etung domain^users   16 Jun  7 20:09 sources/models/src/metal-k8s-1.21 -> ./metal-k8s-1.22
```

It's the `metal-dev` variant as well (or just all metal variants). And we're planning to have the boot settings for all future variants (not limited to `*-k8s-*`). So maybe?:
```suggestion
*Please note that boot settings currently only exist for the baremetal variants and *-k8s-1.23 variants. Boot settings will be added to any future variant introduced after Bottlerocket v1.8.0.
```

Reply:
Thanks, your suggestion is much clearer about expectations for future variants!